src/test/resources/capabilities-bstack-single.yml

```diff
@@ -0,0 +1,38 @@
+testEndpoint: https://bstackdemo.com
+
+driverType: cloudDriver
+
+cloudDriver:
+  hubUrl: https://hub-cloud.browserstack.com/wd/hub
+  user: BROWSERSTACK_USERNAME
+  accessKey: BROWSERSTACK_ACCESSKEY
+  localTunnel:
+    enable: false
+  common_capabilities:
+    project: BrowserStack Demo Repository
+    buildPrefix: browserstack-examples-cucumber-testng
+    capabilities:
+      browserstack.debug: true
+      browserstack.networkLogs: true
+      browserstack.console: debug
+#      browserstack.idleTimeout: 300
```

Comment:
remove. this flag shouldn't be added unless there is a relevant use case.

Reply:
Fixed